CitationStyleLanguagePlugin.php

```diff
@@ -426,14 +426,14 @@ public function getCitation(PKPRequest $request, Submission $submission, string
             $citationData->type = ($this->application === 'ojs2' ? 'article-journal' : 'article');
             $citationData->risType = 'JOUR';
             $citationData->id = $submission->getId();
-            $citationData->title = $publication->getLocalizedFullTitle();
+            $citationData->title = htmlspecialchars_decode($publication->getLocalizedFullTitle());
```

Comment:
Here I just thought about the possibility of a user inserting encoded data in the title (e.g. `&lt;p&gt;`) to abuse. If there's this possibility, a `strip_tags()` would be nice here.